### PR TITLE
nixpkgs-fmt: 1.0.0 -> 1.1.0

### PR DESCRIPTION
**Pull request by SuperSandro2000**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 113939` at 095f8cec run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/ed78c7e4dc4ebd557b9db8e63df13c5b">nixpkgs-fmt</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `fetchpatch`.
Near pkgs/tools/nix/nixpkgs-fmt/default.nix:1:39:
```
  |
1 | { lib, rustPlatform, fetchFromGitHub, fetchpatch }:
  |                                       ^
```
</li>
  </ul>
</details>

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 113939` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>nixpkgs-fmt</li>
  </ul>
</details>

File: pkgs/tools/nix/nixpkgs-fmt/default.nix [hunk 1]
```diff
@@ -1,16 +1,16 @@
 { lib, rustPlatform, fetchFromGitHub, fetchpatch }:
```
Comment (fabaff) [inline pkgs/tools/nix/nixpkgs-fmt/default.nix:1]:
```suggestion
{ lib, rustPlatform, fetchFromGitHub }:
```

See https://github.com/NixOS/nixpkgs/pull/113939#issuecomment-782908253

**Review by fabaff — COMMENTED** [review]
- [x] package built (see https://github.com/NixOS/nixpkgs/pull/113939#issuecomment-782908814)
- [ ] all depending packages build
- [x] tested on x86_64

```bash
...ew/pr-113939]$ ./results/nixpkgs-fmt/bin/nixpkgs-fmt -h
nixpkgs-fmt 1.1.0
Format Nix code
[...]     
```

Looks good to me beside the unused import.

**Review by NeQuissimus — APPROVED** [review]

**Review by davidak — APPROVED** [review]
LGTM

**Comment by davidak** [comment]
Result of `nixpkgs-review pr 113939` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>nixpkgs-fmt</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Next time please give the suggestion a proper name.

**Comment by davidak** [comment]
@SuperSandro2000 what do you mean?

**Comment by SuperSandro2000** [comment]
> @SuperSandro2000 what do you mean?

This commit https://github.com/NixOS/nixpkgs/pull/113939/commits/3ae8bebc9e86b46bcbffc726891aa92a67b0aa29

**Comment by davidak** [comment]
Oh, yes. That should have been squashed.

**Pull request merged**